package.json

```diff
@@ -10,14 +10,16 @@
   },
   "scripts": {
     "start": "nx run cowswap-frontend:serve",
+    "build": "cross-env NODE_OPTIONS=--max-old-space-size=32768 nx build cowswap-frontend",
```

Comment:
I think we no longer need the `--max-old-space` setting, right @shoom3301 ?

Reply:
Happy to remove, but maybe I won't mix in this PR. I can create another one if required. Let me know!